pytx3/pytx3/cli/fetch2.py

```diff
@@ -78,7 +140,11 @@ def _fetch_threat_updates(
         owner: int,
         threat_types: t.List[str],
         additional_tags: t.List[str],
-    ) -> None:
+        next_page: str
+    ):
```

Comment:
Can we set the return type?

Reply:
Function no longer exists; if you look at the newest changes I moved it the fetching into TE.py